index.php

```diff
@@ -104,15 +106,16 @@
     // New filter functionality, searching and listing of requests.
     echo $renderer->render_index_search_controls($context, $categoryid, $courseid, $stateid, $baseurl, $search, $faculty);
 
-    if (capability_checker::can_export_csv()) {
+    // Here we use preference, not capability. 
+    if (((new preferences)->get(preferences::EXPORT_CSV))=='1') {
         $url = clone $baseurl;
         $url->param('download', 'csv');
         echo html_writer::div(
             $OUTPUT->single_button($url, get_string('export_csv', 'local_extension')),
             'local_extension_option_buttons'
         );
     }
-
+    
```

Comment:
Unnecessary extra spaces here, Travis will probably show an error because of that (code style).